apps/server/src/controllers/userControllers/authUsers.ts

```diff
@@ -254,14 +260,15 @@ export const forgotPassword = async (req: Request, res: Response) => {
             otp: otp.toString(),
             tokenExpiry: tokenExpiry.getTime().toString(),
           },
-          { _secret: Env.JWT_SECRET + longString }
+          { _secret: Env.JWT_SECRET + longString, expiresIn }
```

Comment:
I thought we changed longString to secret. I.e : {_secret: Env.JWT_SECRET + secret, expiresIn}
well if we are not using it again, I think its best we don't store it in the DB.